share/spack/gitlab/cloud_pipelines/stacks/developer-tools-darwin/spack.yaml

```diff
@@ -0,0 +1,83 @@
+spack:
+  view: false
+  packages:
+    all:
+      require:
+      - target=aarch64
+  concretizer:
+    unify: true
+    reuse: false
+  definitions:
```

Comment:
@trws I was looking into why there was no lock file to copy to the concrete env dir. I was sort of drawing a blank, but one thing I noticed is that I don't see any messages in the output related to the concretization.  Is it possible your environment is ending up empty? I see here you've got the specs inside `definitions`, but then don't use it in a matrix.  Maybe you can try to remove the `definitions`, and just keep the specs list?

Reply:
That might well be it, I was trying to refactor and maybe got it wrong, thanks for the advice!